analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/SymbolicExecution/SETestContext.cs

```diff
@@ -124,6 +129,8 @@ public unsafe class Sample
     public Sample SampleProperty {{ get; set; }}
     public object AutoProperty {{ get; set; }}
     public object FullProperty {{ get => ObjectField; set => ObjectField = value; }}
+    public static object StaticAutoProperty {{ get; set; }}
+    public static object StaticFullProperty {{ get => StaticObjectField; set => StaticObjectField = value; }}
```

Comment:
I started to fade out the use of `Property` and `StaticProperty` in favor of `AutoProperty`, `FullProperty`, `StaticAutoProperty`, and `StaticFullProperty`. We should expand this effort to all tests and then drop the former properties.